build/storage/python_system_tools/test_platform.py

```diff
@@ -0,0 +1,262 @@
+        )
+        _, rc = self.terminal.execute("docker-compose --version", raise_on_error=False)
+        if rc:
+            logging.info("Setting up docker-compose")
+            for cmd in cmds:
+                out, rc = self.terminal.execute_as_root(cmd)
+                if rc:
+                    logging.error(
+                        f"Setting up docker failed with code {rc} and output {out}"
+                    )
+                    return False
+            logging.info(f"Setup of docker-compose was successful")
+            return True
+        logging.info(f"docker-compose is already installed")
+        return True
+
+    def setup_libguestfs_tools(self) -> bool:
+        """
+        Installs libguestfs-tools for a specific OS
+
+        Returns
+        -------
+        bool
+            True if installation is successful else False
+        """
+
+        program = "libguestfs-tools"
+        if self.os == "Fedora":
+            program += "-c"
+        return self.install(program)
+
+    def change_vmlinuz(self) -> bool:
+        """
+        Changes the mode of /boot/vmlinuz-*
+
+        Returns
+        -------
+        bool
+            True if change is successful else False
+        """
+
+        out, rc = self.terminal.execute_as_root("chmod +r /boot/vmlinuz-*")
+        if rc:
+            logging.error(
+                f"Changing the mode of /boot/vmlinuz-* failed with code {rc} and output {out}"
+            )
+            return False
+        logging.info("Mode of /boot/vmlinuz-* was changed")
+        return True
+
+    def check_security_policies(self) -> bool:
+        """
+        Checks if security policies are disabled
+
+        Returns
+        -------
+        bool
+            True if security policies are disabled else False
+        """
+
+        if self.os == "Fedora":
+            cmd = "setenforce 0"
+        else:
+            cmd = "systemctl stop apparmor"
+        out, rc = self.terminal.execute_as_root(cmd, raise_on_error=False)
+        if "disabled" in out:
+            logging.info("Security policies are disabled")
+            return True
+        logging.error(
+            f"Disabling security policies failed with code {rc} and output {out}"
+        )
+        return False
+
+    def clone_repo(self, branch=None):
+        self.terminal.mkdir(WORKSPACE_PATH)
+        if branch:
+            return self.terminal.execute_as_root(
+                f"git clone -b {branch} https://github.com/ipdk-io/ipdk.git",
+                cwd=WORKSPACE_PATH,
+            )
+        else:
+            return self.terminal.execute_as_root(
+                "git clone https://github.com/ipdk-io/ipdk.git",
+                cwd=WORKSPACE_PATH,
+            )
+
+    def is_installed(self):
+        cmds = ("docker --version", "wget --version", "docker-compose --version")
+        rcs = [self.terminal.execute(cmd)[1] for cmd in cmds]
+        return rcs
+
+    @staticmethod
+    def setup_on_both_machines(
```

Comment:
what means both machines - it's not specify nothing

Reply:
This method has been deleted